open-metadata-implementation/access-services/asset-lineage/asset-lineage-server/src/main/java/org/odpi/openmetadata/accessservices/assetlineage/handlers/ProcessContextHandler.java

```diff
@@ -91,7 +91,8 @@ public ProcessContextHandler(InvalidParameterHandler invalidParameterHandler,
         return checkIfAllRelationshipsExist(userId, process);
     }
 
-    private Map<String, Set<GraphContext>> checkIfAllRelationshipsExist(String userId, EntityDetail entityDetail) throws OCFCheckedExceptionBase {
+    private Map<String, Set<GraphContext>> checkIfAllRelationshipsExist(String userId,
```

Comment:
Should this method have javadoc ? Asking for consistency since other private methods have